src/Controller/DashboardController.php

```diff
@@ -95,12 +154,45 @@ public function ajaxTab(UserInterface $user, $subpage) {
     // Create AJAX Response object.
     $response = new AjaxResponse();
     $response->addCommand(new ContactsTab($subpage, $url->toString()));
-    $response->addCommand(new HtmlCommand('#contacts-tabs-content', $content));
+    $response->addCommand(new HtmlCommand('#contacts-tabs', $content));
 
     // Return ajax response.
     return $response;
   }
 
+  /**
+   * Return the AJAX command for changing tab.
+   *
+   * @param bool|null $manage_mode
+   *   The user we are viewing.
+   *
+   * @return \Drupal\Core\Ajax\AjaxResponse
+   *   The response commands.
+   */
+  public function ajaxManageMode($manage_mode = NULL) {
+    if (is_null($manage_mode)) {
+      // Toggle manage mode.
+      $manage_mode = $this->state()->get('manage_mode');
+    }
+
+    // Get parameters from referrer request.
+    $referrer = $this->request->server->get('HTTP_REFERER');
+    $fake_request = Request::create($referrer);
+
+    /* @var \Drupal\Core\Url $url_object */
+    $url_object = $this->pathValidator->getUrlIfValid($fake_request->getRequestUri());
```

Comment:
Here's a wacky idea... what about a redirect to the referring page?

Reply:
As in, instead of ajax?

Reply:
Let's ignore this for now!